ocp-build-data-validator/tests/test_format.py

```diff
@@ -12,7 +12,13 @@ def test_invalid_yaml(self):
         """
         (parsed, err) = format.validate(invalid_yaml)
         self.assertIsNone(parsed)
-        self.assertIn("did not find expected key", err)
+        # Check for different possible error messages from different YAML parser versions
```

Comment:
I am wondering, does the parser not raise an Exception when encountering invalid input? We could use `assertRaises` in that case